AppConnectSwift/OnePageFormViewController.swift

```diff
@@ -107,40 +107,40 @@ class OnePageFormViewController: UIViewController {
         // you don't know in advance what the fields are going to be,
         // look at MultiPageFormViewController.
         let sequencer = MDStepSequencer(form: self.form)
-        sequencer.start()
+        sequencer?.start()
```

Comment:
Would it be worth unwrapping or guarding the `sequencer` constant so we don't have to jump through hoops in the rest of this method?